client/src/auth/index.tsx

```diff
@@ -2,41 +2,40 @@ import React from "react";
 
 import { PageContentContainer } from "../ui/atoms/page-content";
 
+import "./index.scss";
+
 const SignInApp = React.lazy(() => import("./sign-in"));
 const SignUpApp = React.lazy(() => import("./sign-up"));
 
 function Container({
-  pageTitle,
   children,
   className,
 }: {
-  pageTitle: string;
   children: React.ReactNode;
   className: string;
 }) {
   const isServer = typeof window === "undefined";
+  const pageTitle = "Sign in to MDN Web Docs";
   React.useEffect(() => {
     document.title = pageTitle;
   }, [pageTitle]);
```

Comment:
If the page title is hardcoded it can't change so it doesn't make sense as a dependency. To fix, change it to:
```suggestion
  }, []);
```